types.go

```diff
@@ -80,9 +81,14 @@ type VoterInfo struct {
 	Proxy             AccountName   `json:"proxy"`
 	Producers         []AccountName `json:"producers"`
 	Staked            Int64         `json:"staked"`
-	LastVoteWeight    Float64       `json:"last_vote_weight"`
-	ProxiedVoteWeight Float64       `json:"proxied_vote_weight"`
+	LastVoteWeight    Decimal       `json:"last_vote_weight"`
+	ProxiedVoteWeight Decimal       `json:"proxied_vote_weight"`
```

Comment:
I've review the `Decimal` type below, I'm quite unsure about this change. If the API decided to use a `Float64`, then probably that precision was not of the utmost importance, because float are know to be imprecise. They would have used a `Int` directly with an amount of decimals to represent precise number.

It will be also a breaking change for users because they will not be able to cast to `float64` directly and it seems no conversion exists. I'm not against breaking changes but in this case, I do not see the reason to do it.

Reply:
Yes, even I don't want to make ambreaking change, but want to ask your opinion.
You implemented big number presentation yourself but now native-like `big.Int` is broadly used.
Between replacing to big.Int or keeping your implmentation, which do you prefer?

+ in JSON assertion, wish not to compare the number but impossible
Let me try manual typing one by one

Reply:
By https://github.com/eoscanada/eos-go/pull/191#discussion_r1016782712 , the the type will be removed

Reply:
> You implemented big number presentation yourself but now native-like big.Int is broadly used

Not sure what you are referring to exactly, of which type you are talking about exactly?

> Plus, in JSON assertion, wish not to compare the number but impossible

Agree, it's harder to deal with full JSON assertion for some numbers. I would not go manual typing neither, maybe just change the JSON value to be 0.0 or something that always equivalent. The goal is to ensure the representation of the response is good, not that much that it's precise.

And like mentioned below, reference contracts uses `double` (https://github.com/AntelopeIO/reference-contracts/blob/main/contracts/eosio.system/include/eosio.system/eosio.system.hpp#L302) so I don't see a good reason to switch the type here, `float64` in Golang is a `double` in C++.

Please revert the type change and remove `Decimal/Int` types for now until there is a use for them.

Reply:
By https://github.com/eoscanada/eos-go/pull/191#discussion_r1016782712 , the type will be removed